### PR TITLE
add options to enable onesync infinity

### PR DESCRIPTION
**Pull request by Holfz**
Since onesync infinity/big mode is now avaliable for all "Patreon FiveM Element Club Platinum" so is it time for adding enable/disable button and setting for infinity?

### CODE REVIEW

File: src/webroutes/diagnostics.js [hunk 1]
```diff
@@ -188,6 +188,7 @@ async function getFXServerData(){
             versionWarning: versionWarning || '',
             resources: infoData.resources.length,
             onesync: (infoData.vars && infoData.vars.onesync_enabled === 'true')? 'enabled' : 'disabled',
+            //infinity: (infoData.vars && infoData.vars.infinity_enabled === 'true')? 'enabled' : 'disabled',
```
Comment (Holfz) [inline src/webroutes/diagnostics.js:191]: Info.json didn't provide a vars for infinity yet. So i commented this out

File: web/settings.html [hunk 2]
```diff
@@ -180,6 +180,19 @@ <h2>Settings</h2>
...
 
+                        <div class="form-group row">
+                            <label for="frmFXServer-infinity" class="col-sm-3 col-form-label">OneSync Infinity</label>
+                            <div class="col-sm-9">
+                                <label class="switch switch-label switch-pill switch-success fix-pill-form">
+                                    <input class="switch-input" type="checkbox" id="frmFXServer-infinity" {{fxserver.infinity}} {{disableWrite}}>
+                                    <span class="switch-slider" data-checked="On" data-unchecked="Off"></span>
+                                </label>
+                                <span class="form-text text-muted">
+                                    Required if you server has more than 128 slots.
```
Comment (FlawwsX) [inline web/settings.html]: Change it **from**: "Required if you server has more than 128 slots."
**To**: "Required if your server has more than 128 slots and supports Onesync Infinity."

Comment (Holfz) [reply web/settings.html]: Yeah. Done!

File: web/settings.html [hunk 3]
```diff
@@ -175,7 +175,7 @@ <h2>Settings</h2>
                                     <span class="switch-slider" data-checked="On" data-unchecked="Off"></span>
                                 </label>
                                 <span class="form-text text-muted">
-                                    Required if you server has more than 32 slots.
+                                    Required if your server has more than 32 slots.
```
Comment (tabarra) [inline web/settings.html:178]: FINALLY someone noticed this xD

**Review by tabarra — APPROVED** [review]

**Comment by tabarra** [comment]
Thank you for the PR.  
As I said in discord, I'll check with FiveM if we still need those flags, specifically for infinity, and if that's the case I will merge this PR.

**Comment by NeOMakinG** [comment]
What's up @tabarra ? Some PR's looks cool and would require some attention :)

**Comment by tabarra** [comment]
@NeOMakinG I'm aware, but currently I'm working on the conversion branch that is already 86 commits ahead of master and is largely incompatible.  
Don't worry, this project is far from abandoned.

**Comment by NeOMakinG** [comment]
Nice to hear that, I'm gonna contribute in next weeks probably, just came in the FiveM world because of getting hired for a project by some friends, keep the good work

**Comment by tabarra** [comment]
Already present in txAdmin v2.  
Thanks for your contribution tho.

**Pull request closed without merging**